### PR TITLE
Fixed missing where clause

### PR DESCRIPTION
**Pull request by eikaramba**
please update immediately, as this breaks current feathers-sequelize if eager-loading is used (luckily quite a new feature anyway). Sry for that :/

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -66,10 +66,12 @@ class Service {
     let promise;
 
     if (params.sequelize && params.sequelize.include) { // If eager-loading is used, we need to use the find method
+      params.query = params.query || {};
```
Comment (daffl) [inline src/index.js]: We don't want to mutate the object we get passed as the query.

**Comment by daffl** [comment]
Can we add some tests for this feature to make sure everything works as expected?

**Comment by eikaramba** [comment]
Unforunately i'm not so confident in writing tests and in this case it would be impossible right? Because you need a database to test against ....... However, one could maybe add a local .sqlite file as a database and test against that.

**Comment by daffl** [comment]
That is what the existing tests are already doing, would be nice to look into it at one point. Fix released as v1.4.2.

**Pull request merged**